### ISSUE TITLE
Swift version

### ISSUE THREAD

**Issue by Koshub** [mysterioustrousers/MTDates#57]
Perfect pod, will you do the Swift version?

**Comment by pwightman** [comment]
Likely won't rewrite in Swift, this library wouldn't benefit much from Swift-specific features. But it may be good to put in `nonnull` and `nullable` attributes on params/return types so it's more Swift-friendly.

**Comment by atomkirk** [comment]
Yeah im going to write a date library in swift but it dont resemble the structure of this one at all. This one has a lot of bad design choices and like @pwightman said, a lot of swift features can make this much easier and intuitive to use, as well as much more readable.

**Comment by Koshub** [comment]
@atomkirk,  I meant rewrite with Swift features

**Comment by truemetal** [comment]
If there would be native `Date` support so that you don't have to from `NSDate` that would work perfectly
